.circleci/config.yml

```diff
@@ -108,6 +111,7 @@ jobs:
                 name: Install Dash
                 command: |
                     . venv/bin/activate
+                    pip install --upgrade pip
```

Comment:
Here you're in the same venv as [line 102](https://github.com/plotly/dash-bio/pull/570/files#diff-78a8a19706dbd2a4425dd72bdab0502ed7a2cef16365ab7030a5a0588927bf47R102) so this should be redundant